### ISSUE TITLE
Real time update fails silently when using pt::text

### ISSUE THREAD

**Issue by kbrabrand** [sanity-io/next-sanity#18]
When using the latest version (`next-sanity@0.2.0`) the `usePreviewSubscription` hook does not return live draft data if the query utilizes the [`pt::text`](https://www.sanity.io/docs/groq-functions#1467558a0339) function.

The query is successful and returns fresh data in vision and also works fine through the client when building the next page. The hook however returns the data passed to the `initialData` property of the hook. I can see that the subscription is set up correctly, and the page is re-rendered when I do changes in sanity studio, but the data from the preview subscription hook is still returning the initialData. No errors are logged to the console.

This will return initialData 👇 

```
const { data } = usePreviewSubscription(groq`
  *[_type == 'frontpage']{
    ...,
    content[]{
      ...,
      articles[]->{
        _id,
        slug,
        title,
        "preamble": pt::text(preamble)
      }
    }
  }
`, { initialData, enabled })
```

...while this one will return live updates once the dataset is downloaded and listener is set up👇 

```
const { data } = usePreviewSubscription(groq`
  *[_type == 'frontpage']{
    ...,
    content[]{
      ...,
      articles[]->{
        _id,
        slug,
        title,
        "preamble": "blabla" // 👈 no pt::text(...)
      }
    }
  }
`, { initialData, enabled })
```

**Comment by judofyr** [comment]
This will be fixed once we release the next version of groq-js: sanity-io/groq-js#41

**Comment by judofyr** [comment]
And then we need this one: https://github.com/sanity-io/groq-store/pull/3

**Comment by judofyr** [comment]
And once this is merged and released it should start finally working: https://github.com/sanity-io/next-sanity/pull/26

**Comment by judofyr** [comment]
This should now be fixed in version 0.3.0.
